tests/test-steering-render.sh

```
@@ -1481,6 +1481,81 @@ else
+  && pass "skills reference the installed principles by absolute path" \
+  || fail "skill cross-references were not rewritten"
+grep -q 'touchstone-git-workflow/SKILL.md' "$HS/.claude/skills/.touchstone-installed" \
+  && grep -q 'memory-audit/agents/openai.yaml' "$HS/.claude/skills/.touchstone-installed" \
+  && pass "skills are recorded in their own manifest with relative paths" \
+  || fail "skills manifest incomplete: $(cat "$HS/.claude/skills/.touchstone-installed")"
+bash "$INSTALL" check --home "$HS" >/dev/null 2>&1 && pass "check passes after a skills install" || fail "check fails right after install"
+printf 'edited\n' >>"$HS/.claude/skills/touchstone-pre-impl/SKILL.md"
+bash "$INSTALL" check --home "$HS" >"$TMP_DIR/hs.check" 2>&1 && fail "an edited skill was reported current" \
+  || { grep -q 'DRIFT: .claude/skills' "$TMP_DIR/hs.check" && pass "an edited skill is drift" || fail "drift did not name the skills set: $(cat "$TMP_DIR/hs.check")"; }
+bash "$INSTALL" install --home "$HS" >"$TMP_DIR/hs.out" 2>&1
+grep -q 'preserved: touchstone-pre-impl/SKILL.md' "$TMP_DIR/hs.out" && pass "an edited skill is preserved before reinstall" || fail "edited skill not preserved"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)